ff/serving/provider/sql.go

```diff
@@ -35,10 +37,13 @@ type OfflineTableQueries interface {
 	registerResources(db *sql.DB, tableName string, schema ResourceSchema, timestamp bool) error
 	primaryTableRegister(tableName string, sourceName string) string
 	primaryTableCreate(name string, columnString string) string
-	getColumnNames() string
+	getColumnNames(db *sql.DB, tableName string) (*sql.Rows, error)
```

Comment:
Its because postgres and snowflake execute this query differently so the database pointer is passed through instead of the function just returning a string

Reply:
Function name is confusing. Does it return an iterator on rows that represent column names?

Reply:
Saw your other comment and I understand. Does sql.Rows have a length function?

Reply:
can't you return an []string with all the queries instead? I still don't undersatnd why you have to pass in the db

Reply:
Will have it return strings instead. Its because snowflake needs to bind the table name and postgres doesnt so they need specific implementations